modular_skyrat/code/_DEFINES/medical_defines.dm

```diff
@@ -0,0 +1,3 @@
+#define PAINKILLER_MINERSSALVE	"PAINKILLER - MINERS SALVE"
+#define PAINKILLER_MORPHINE 	"PAINKILLER - MORPHINE"
```

Comment:
second define?

Reply:
This file is no longer included and was deleted.